Code/Framework/AzCore/AzCore/Serialization/DataPatch.cpp

```diff
@@ -144,29 +144,32 @@ namespace AZ
     void DataNodeTree::Build(const void* rootClassPtr, const Uuid& rootClassId)
     {
         AZ_PROFILE_FUNCTION(AzCore);
-
+        using namespace AZ::Edit;
```

Comment:
The `using namespace AZ::Edit` isn't needed.
Nothing uses the `AZ::Edit` namespace in this function.